doc/dev/crimson.rst

```diff
@@ -0,0 +1,143 @@
+=======
+crimson
+=======
+
+crimson is the code name of crimson-osd, which the next generation ceph-osd. It
+targets fast storage devices, like NVMe storage, for better performance.
+
+
+daemonize
+---------
+
+Unlike ``ceph-osd``, ``crimson-osd`` does daemonize itself even if the
+``daemonize`` option is enabled. Because, to read this option, ``crimson-osd``
```

Comment:
> ``crimson-osd`` does daemonize itself even if the``daemonize`` option is enabled.

FYI, the above sentence confuses me. s/enabled/not enabled/?

Reply:
yeah, it is confusing. i put `-f` but changed my mind and used `daemonize` instead. but failed to update the first part of the sentence.